pytest/lib/cluster.py

```diff
@@ -362,10 +374,25 @@ def spin_up_node(config, near_root, node_dir, ordinal, boot_key, boot_addr, blac
     return node
 
 
+def connect_to_mocknet(config):
```

Comment:
what is mocknet?

Reply:
It's an arbitrary network running that is not spun up by the test, against which a set of tests will be executed.
The actual nodes of the network are picked up from a local config